examples/hellocluster-yaml/config.yaml

```
@@ -0,0 +1,22 @@
+# The cluster to run on
+cluster: kislurm
+
+# Everything that goes into the SBATCH preamble goes here
+job:
+  partition: testdlc_gpu-rtx2080
```

Anonimize with something like:
```suggestion
  partition: YOURPARTITION
```